src/components/sections/footer.tsx

```
@@ -1,9 +1,119 @@
+import {
+  baseOptions,
+  linkItems,
+  postsPerPage,
+  title,
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Safe fix: Remove the unused import.


(lint/correctness/noUnusedImports)